### PR TITLE
changing default editor to a normal persons editor

### PR DESCRIPTION
**Pull request by danbaehr**
lets take it to the bigs with this change

### CODE REVIEW

**Comment by rhrhunter** [comment]
I see that even though this has passed the travis tests, I don't think the world is ready for changing the default editor from emacs to vim.

File: etc/bash/env.bash [hunk 1]
```diff
@@ -17,7 +17,7 @@ set -o ignoreeof
 export TERM=xterm
 
 # sane people use this editor
-export EDITOR="emacs"
+export EDITOR="vim"
```
Comment (marcusmolchany) [inline etc/bash/env.bash:20]: curious to know why this part didn't make it through code review?

**Comment by rhrhunter** [comment]
Am I going to have to lock this conversation down?

**Comment by marcusmolchany** [comment]
might i propose

``` bash
alias emacs="echo 'no, think about what you've just done.'; sleep 5; vim"
```

**Comment by rhrhunter** [comment]
Perhaps, but we would need this alias as well.

``` bash
alias vim="reboot"
```

And that would obviously go into `etc/bash/alias.bash`, not the `etc/bash/env.bash` file.

**Comment by marcusmolchany** [comment]
ok, opened issue #4

**Comment by rhrhunter** [comment]
goddamnit.

**Comment by danielalmond** [comment]
why not just end the profile script with sudo -i pompous_asshole?

**Comment by rhrhunter** [comment]
Since this is a he said she said issue, i think the only change i will accept for this is for the Makefile to be updated to query the user on their editor of choice during the installation process. You would also need to provide a mechanism for the make to force it to choose a default of emacs so that the travis CI tests will be able to still run without requiring user intervention.

**Pull request closed without merging**